credentials/ssocreds/sso_credentials_provider.go

```diff
@@ -45,6 +45,10 @@ type Options struct {
 	// If custom cached token filepath is used, the Provider's startUrl
 	// parameter will be ignored.
 	CachedTokenFilepath string
+
+	// Used by the SSOCredentialProvider if a token configuration
+	// profile is used in the shared config
+	TokenProvider *SSOTokenProvider
```

Comment:
Probably should prefix this member with `SSO` in case there are other kinds of `TokenProvider` in the future. `CachedTokenFilepath` probably should of been prefixed with `SSO` as well, but that ship has passed.

Reply:
gotcha, just made this change with a new commit